test/integration/map/MapProxyTest.js

```diff
@@ -559,26 +559,28 @@ describe('MapProxyTest', function () {
                     });
             });
 
-            it('addEntryListener on key, entryAdded fires because predicate returns true for that entry, inlVal=yes', function (done) {
-                const listenerObject = {
-                    added: function (entryEvent) {
-                        try {
-                            expect(entryEvent.name).to.equal('test');
-                            expect(entryEvent.key).to.equal('key10');
-                            expect(entryEvent.value).to.equal('val10');
-                            expect(entryEvent.mergingValue).to.be.null;
-                            expect(entryEvent.member).to.not.be.null;
-                            done();
-                        } catch (err) {
-                            done(err);
+            it('addEntryListener on key, entryAdded fires because predicate returns true for that entry, inlVal=yes',
+            function (done) {
```

Comment:
nit: I'd prefer to add padding here:
```suggestion
                function (done) {
```